### PR TITLE
Update installation instructions in README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Laurent2916**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,12 +56,10 @@ ______________________________________________________________________
 
 ## Installation
 
-The current recommended way to install Refiners is from source using [Rye](https://rye-up.com/):
+The current recommended way to install Refiners is from source:
 
 ```bash
-git clone "git@github.com:finegrain-ai/refiners.git"
```
Comment (deltheil) [inline README.md:62]: I would keep this. And just add:

Alternatively:

```bash
pip install git+https://github.com/finegrain-ai/refiners.git
```

Comment (deltheil) [reply README.md:62]: @Laurent2916 wdyt?

Comment (Laurent2916) [reply README.md:62]: I feel like the whole "rye setup" is more "dev oriented" than "user oriented".
wdyt about simply adding a reference to the CONTRIBUTING.md if people want to setup a dev env ?

Comment (deltheil) [reply README.md:62]: True. But https://refine.rs/getting-started/recommended/ is rye-centric.

Comment (deltheil) [reply README.md:62]: Anyway, let's rebase and use one or the other way.

Comment (catwell) [reply README.md:62]: The recommended option on the website is Rye because you needed to convert the weights anyway. We could revisit this.

**Review by deltheil — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Laurent2916) [reply README.md:62]: Ah right, there is still the weight conversion. I agree we should probably revisit this before modifying the README.

**Pull request closed without merging**